### ISSUE TITLE
ringActionState stays true

### ISSUE THREAD

**Issue by dennisoderwald** [bluewalk/NukiBridge2Mqtt#47]
I noticed that the topic "ring-action-state" remains True after a ring and does not set back to False.

The official documentation of the bridge says:

> Flag indicating if a ring-action is currently
> occuring or not (reset after 30 seconds)

Would it be possible to adopt this? Thanks so far for your work!

**Comment by bluewalk** [comment]
It should be parsed by the callback from the bridge according to https://developer.nuki.io/uploads/short-url/5tLp76dEJ1RYfkSUKxFTUSeyJNg.pdf

Would it be possible to set logging to Debug and post the callback for?
I suspect that the "reset after 30 seconds" doesn't trigger a callback thus never updating the RingActionState.

**Comment by dennisoderwald** [comment]
This is correct, no further callback follows. You could either reset the status after 30 seconds or call up and update the device info about 30 seconds later.
